db/seeds.rb

```diff
@@ -7,14 +7,16 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 require 'faker'
 
+raise "You do not have a Redis server running" unless Sidekiq.redis { |r| r.echo "Alive" }
```

Comment:
For some reason, Repos fail to save if there's no Redis server running, despite the test queue adapter.